### ISSUE TITLE
[CoL] Crash on load after updating to Quark 2.4-280

### ISSUE THREAD

**Issue by drakray** [MinecraftModDevelopment/Extra-Golems#64]
https://pastebin.com/TXD5dCb0

MC 1.16.4
Forge 35.1.13
[Quark-r2.4-280](https://www.curseforge.com/minecraft/mc-mods/quark/files/3139878)
extragolems-11.1.9

<details>
<summary>Crash Snippet</summary>

```
Thread: Render thread
Stacktrace:
	at java.lang.ClassLoader.findClass(Unknown Source) ~[?:1.8.0_271] {}
-- MOD golems --
Details:
	Caused by 0: java.lang.NoClassDefFoundError: vazkii/quark/world/module/NewStoneTypesModule
		at com.mcmoddev.golems_quark.QuarkGolemsEntities.initEntityTypes(QuarkGolemsEntities.java:87) ~[golems:11.1.9] {re:classloading}
		at com.mcmoddev.golems.integration.AddonLoader.initEntityTypes(AddonLoader.java:38) ~[golems:11.1.9] {re:classloading}
		at com.mcmoddev.golems.main.ExtraGolems.<init>(ExtraGolems.java:60) ~[golems:11.1.9] {re:classloading}
		at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_271] {}
		at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source) ~[?:1.8.0_271] {}
		at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source) ~[?:1.8.0_271] {}
		at java.lang.reflect.Constructor.newInstance(Unknown Source) ~[?:1.8.0_271] {}
		at java.lang.Class.newInstance(Unknown Source) ~[?:1.8.0_271] {}
		at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:81) ~[forge:35.1] {re:classloading}
		at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:120) ~[forge:?] {re:classloading}
		at java.util.concurrent.CompletableFuture$AsyncRun.run(Unknown Source) ~[?:1.8.0_271] {}
		at java.util.concurrent.CompletableFuture$AsyncRun.exec(Unknown Source) ~[?:1.8.0_271] {}
		at java.util.concurrent.ForkJoinTask.doExec(Unknown Source) ~[?:1.8.0_271] {}
		at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(Unknown Source) ~[?:1.8.0_271] {}
		at java.util.concurrent.ForkJoinPool.runWorker(Unknown Source) ~[?:1.8.0_271] {}
		at java.util.concurrent.ForkJoinWorkerThread.run(Unknown Source) ~[?:1.8.0_271] {}
```

</details>

**Comment by BugmanBugman** [comment]
can confirm

**Comment by BedrockLegends** [comment]
Had the same issue

**Comment by MarioMario24680** [comment]
Same here. The previous build, r2.4-279, was stable and did not cause any issues with the latest version of the Extra Golems mod for 1.16.4 (version 11.1.9). However, this new build, r2.4-280, seems to cause a crash, both client-side ([crash-2020-12-17_02.16.03-fml.txt](https://github.com/MinecraftModDevelopmentMods/Extra-Golems/files/5707882/crash-2020-12-17_02.16.03-fml.txt)) and server-side ([crash-2020-12-17_02.13.52-fml.txt](https://github.com/MinecraftModDevelopmentMods/Extra-Golems/files/5707868/crash-2020-12-17_02.13.52-fml.txt)), whenever I start up modded (Forge 35.1.10, no Fabric) Minecraft [1.16.4.] The creator of Quark said in this thread that some files were moved around in 280, and it looks like extra golems is still trying to access the old locations:
**https://github.com/Vazkii/Quark/issues/2734**

So the creators of Extra Golems need to update their mod in order for it to be compatible with Quark again, since the creator of Quark has no intention of changing Quark back to the way it used to be.

**Comment by skyjay1** [comment]
Fixed in Extra Golems 11.2.0
